flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/combined/CombinedNtProperties.java

```diff
@@ -0,0 +1,66 @@
+package com.flash3388.frc.nt.combined;
+
+import edu.wpi.first.networktables.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CombinedNtProperties {
```

Comment:
Weird name... call it something like `NtTable`

Reply:
k